### PR TITLE
AT 04.09.02 | Folder > Move Freestyle project in folder

### PR DESCRIPTION
**Pull request by MaksPt**
https://trello.com/c/yvgOT9Zv/309-at-040902-folder-move-freestyle-project-in-folder
Created test:
testMoveFreestyleProjectInFolderUsingDropDownMenu()

### CODE REVIEW

**Review by snysmymric — APPROVED** [review]
Ok

File: src/test/java/FolderTest.java [hunk 1]
```diff
@@ -223,4 +224,31 @@ public void testDeleteFolder2() {
...
+        createProjectFromDashboard(FOLDER, folderName);
+        getDriver().findElement(DASHBOARD).click();
+        createProjectFromDashboard(FREESTYLE_PROJECT, freestyleProjectName);
+        getDriver().findElement(DASHBOARD).click();
+
+        Actions action = new Actions(getDriver());
+        action
+                .moveToElement(getDriver().findElement(By.linkText(freestyleProjectName)))
+                .moveToElement(getDriver().findElement(By.xpath("//tr[@id = 'job_" + freestyleProjectName + "']//td/a/button")))
+                .click()
```
Comment (UmidaKaharova) [inline src/test/java/FolderTest.java:242]: I believe you may fit click().build().perform()  into one line

File: src/test/java/FolderTest.java [hunk 2]
```diff
@@ -223,4 +224,31 @@ public void testDeleteFolder2() {
...
+                .build().perform();
+        getDriver().findElement(By.xpath("//span[contains(text(),'Move')]")).click();
+
```
Comment (UmidaKaharova) [inline src/test/java/FolderTest.java:245]: unnecessary empty line

File: src/test/java/FolderTest.java [hunk 3]
```diff
@@ -223,4 +224,31 @@ public void testDeleteFolder2() {
...
+        getDriver().findElement(By.xpath("//option[@value='/" + folderName + "']")).click();
+        getDriver().findElement(By.xpath("//button[@type='submit']")).click();
+        getDriver().findElement(DASHBOARD).click();
+
```
Comment (UmidaKaharova) [inline src/test/java/FolderTest.java:249]: unnecessary empty line

**Review by UmidaKaharova — COMMENTED** [review]
small remarks, otherwise LGFM

File: src/test/java/FolderTest.java (hunk 2 above)
Comment (MaksPt) [reply src/test/java/FolderTest.java:245]: The empty line separates the logical blocks of the test in my opinion.

File: src/test/java/FolderTest.java (hunk 3 above)
Comment (MaksPt) [reply src/test/java/FolderTest.java:249]: The empty line separates the logical blocks of the test in my opinion.

File: src/test/java/FolderTest.java (hunk 1 above)
Comment (MaksPt) [reply src/test/java/FolderTest.java:242]: Maybe yes, but maybe this is "vkysovshina":) Thank you for review.

**Review by AndreiShupayeu — APPROVED** [review]

**Pull request merged**